useful/template.environment.py

```diff
@@ -0,0 +1,22 @@
+GAME_NAME = "Adventure Land"
+APPENGINE_ID = "twodimensionalgame"
+DOMAIN_NAME = "adventure.land"
+#IMPORTANT: SPECIAL PAGE RULES ARE NEEDED: https://dash.cloudflare.com/b6f5a13bded5fdd273e4a1cd3777162d/adventure.land/page-rules - uss1 / eus1 was best
+IP_TO_SUBDOMAIN = {
```

Comment:
This does not seem to be used anywhere

Reply:
it's used in `config.py` it's from the `environment` import and IP_TO_SUBDOMAIN was used for `if domain.https_mode: ip="%s.%s"%(ip_to_subdomain.get(ip,ip),live_domain)` perhaps i've messed up a merge conflict. e.g. `ip_to_subdomain` should have been initialized to the value from environment

Reply:
Ok well in your whole branch it seems missing, right now!

Reply:
If you mean `IP_TO_SUBDOMAIN` then yes that might be missing.

However  `APPENGINE_ID` and `DOMAIN_NAME` are used in `config.py` from `environment.py`

The file `template.environment.py` is used to as a template for `environment.py`

Reply:
Yes, github adds context above what I selected but I'm thinking about `IP_TO_SUBDOMAIN` specifically, indeed.